quantum/pointing_device/pointing_device_drivers.c

```diff
@@ -107,8 +107,10 @@ void cirque_pinnacle_enable_cursor_glide(bool enable) {
     cursor_glide_enable = enable;
 }
 
-void cirque_pinnacle_configure_cursor_glide(float trigger_px) {
+void cirque_pinnacle_configure_cursor_glide(float trigger_px, float coef, float interval) {
```

Comment:
I kept these as floats. I noticed some discussion in the original PR about `float` however it was not clear to me if these should be something different (like `uint16_t`). 

Happy to adjust these as desired!